cumulusci.yml

```diff
@@ -388,6 +388,16 @@ tasks:
                 %%%NAMESPACE%%%Error_Settings__c errSettings = %%%NAMESPACE%%%Error_Settings__c.getOrgDefaults();
                 errSettings.%%%NAMESPACE%%%OverrideFeature_PilotEnabled__c = true;
                 upsert errSettings;
+    
+    execute_rd2_data_migration:
```

Comment:
@codexmarcel , @force2b : can you please review changes?

Reply:
Using a task for this is optional. It's helpful when there may be a need to run a data migration completely independent of the enablement process, but I don't necessarily think that would ever happen. I think it might be better to just have this as part of the flow without the independent task. 
@codexmarcel Are you ok with that?

Reply:
The `execute_rd2_data_migration` has been removed and replaced with `execute_anon` to run the data migration batch.